charts/op-scim-bridge/values.yaml

```diff
@@ -1,7 +1,7 @@
 # scim configuration options
 scim:
   # name sets the name used for the various constructs used by the SCIM bridge setup
-  name: "{{ .Chart.Name }}"
+  name: "{{ .Release.Name }}"
```

Comment:
@grellyd, @alicethorne-ab: This is good to go, but what do you think about appending "-bridge" or "-scim-bridge" to this value, i.e. similar to "-redis-master" below.

The reason I ask is that if you call your release "my-release" the application deployment/pod name doesn't really indicate that this is the (scim) bridge part.

Reply:
That sounds like a good idea to me. Could you demonstrate what the end result would look like in `kubectl get pods`?

Reply:
Example names incoming...

Reply:
These are the pod names currently (without appending anything to the name):
<img width="1511" alt="image" src="https://user-images.githubusercontent.com/11049609/179259934-f1ede084-2c20-43ea-a770-c8a1fff4586a.png">

This is what the pod names are with appending "-scim-bridge":
<img width="1489" alt="image" src="https://user-images.githubusercontent.com/11049609/179262091-91162993-d40e-4b99-ba0e-f84b4f1f4826.png">

Thoughts, opinions, insults? 😄

Reply:
I'd prefer the later, more verbose, naming. However keep in mind that the name might be referred elsewhere and appending fixed strings in one place might not be the best solution... 

redis might use the fullName helper function from here: https://github.com/bitnami/charts/blob/master/bitnami/common/templates/_names.tpl#L21 ?

Reply:
Good point @pniederlag. Thanks for linking to the redis helper func. I actually like the redis naming scheme and helper functions a lot. Let me try using it and seeing what the results look like.